packages/kbn-es/src/cli_commands/serverless.ts

```diff
@@ -28,9 +27,8 @@ export const serverless: Command = {
     return dedent`
     Options:
 
-      --tag               Image tag of ESS to run from ${SERVERLESS_REPO} [default: ${SERVERLESS_TAG}]
-      --image             Full path of ESS image to run, has precedence over tag. [default: ${SERVERLESS_IMG}]
-      
+      --tag               Image tag of ESS to run from ${ES_SERVERLESS_REPO_ELASTICSEARCH}
```

Comment:
I removed the default, because it would be a bit deceitful. There's in fact no default tag, but a default image.